aws-amp-integration.md

```diff
@@ -1,70 +1,154 @@
+You can use the following AWS CLI command to create a new AMP workspace:
 
+```bash
+export AWS_REGION=<YOUR-AWS-REGION>
+aws amp create-workspace --alias kubecost-amp --region $AWS_REGION
 ```
-# replace the namespace and filename if needed
-$ helm upgrade --install kubecost kubecost/cost-analyzer --namespace kubecost -f ./values.yaml
+Example output:
+
+```json
+{
+    "arn": "arn:aws:aps:us-west-2:xxxxxxxxxxxxx:workspace/${AMP_WORKSPACE_ID}",
+    "status": {
+        "statusCode": "CREATING"
+    },
+    "tags": {},
+    "workspaceId": "${AMP_WORKSPACE_ID}"
+}
 ```
 
-Verify that Kubecost is using the Prometheus server configured to remote write metrics to AMP rather than Kubecost's default Prometheus installation by forwarding Kubecost to a local port and querying the `/api` endpoint:
+The workspace should be created in a few seconds. You can log in to [AWS AMP console](https://console.aws.amazon.com/prometheus/) to retrieve more information. You need to set `$REMOTEWRITEURL` and `$QUERYURL` for using in the integration with Kubecost later as follows:
 
-```
-# replace `kubecost` and `deployment/kubecost-cost-analyzer` if needed
-$ kubectl port-forward --namespace kubecost deployment/kubecost-cost-analyzer 9090
+REMOTEWRITEURL="https://aps-workspaces.us-west-2.amazonaws.com/workspaces/${AMP_WORKSPACE_ID}/api/v1/remote_write"
+QUERYURL="http://localhost:8005/workspaces/${AMP_WORKSPACE_ID}"
+
+### Install Kubecost with the default values
+
+#### Prerequisites
+- Install the following tools: [Helm 3.9+](https://helm.sh/docs/intro/install/), [kubectl](https://kubernetes.io/docs/tasks/tools/), and optionally [eksctl](https://eksctl.io/) and [AWS CLI](https://aws.amazon.com/cli/).
+- You have access to an [Amazon EKS cluster](https://aws.amazon.com/eks/).
 
-# in another Terminal window
-$ curl http://localhost:9090/api/
+#### Installation
+
+Run the following command to install Kubecost from Amazon ECR Public Gallery:
+
+```bash
+helm upgrade -i kubecost \
+oci://public.ecr.aws/kubecost/cost-analyzer --version <$VERSION> \
+--namespace kubecost --create-namespace \
+-f https://tinyurl.com/kubecost-amazon-eks
 ```
+### Set up IAM role for Kubecost service account (IRSA):
 
-The first line returned by the `curl` command should contain your `fqdn` parameter URL, like so:
+These following commands help to automate the following tasks:
+- Create an IAM role with the AWS managed IAM policy and trusted policy for the following service accounts: `kubecost-cost-analyzer`, `kubecost-prometheus-server`.
+- Modify current K8s service accounts with annotation to attach new IAM role.
+
+> **Note**: remember to replace `<YOUR_CLUSTER_NAME>` and `<AWS_REGION>` with your desired values
 
 ```
-Using Prometheus at http://prometheus-server.prometheus.svc.cluster.local.
+export YOUR_CLUSTER_NAME=<YOUR-CLUSTER-NAME>
+export AWS_REGION=<YOUR-AWS-REGION>
+
+kubectl create ns kubecost
```

Comment:
shouldn't this have already happened in previous step?

Reply:
I include it to ensure the command will work even when customer switch between the terminals. If you think that it is unnecessary, I can remove it before merging.

Reply:
Btw, I removed them to make it align with AWS doc and blog post. Thank you Webb !

Reply:
Feels like it's fine either way, just wanted to call out in case it caused an error